examples/soupbintcp-server/pom.xml

```diff
@@ -0,0 +1,56 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.paritytrading.nassau</groupId>
+    <artifactId>nassau-parent</artifactId>
+    <version>0.13.1-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>nassau-soupbintcp-server</artifactId>
+
+  <name>Nassau SoupBinTCP Server</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>nassau-core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.hdrhistogram</groupId>
+      <artifactId>HdrHistogram</artifactId>
+    </dependency>
```

Comment:
I thought I mentioned about this earlier, but it looks like I forgot: this library is actually not used on the server side, so this dependency can be removed.

Reply:
Done